### ISSUE TITLE
Feature request(?): Allow for using `check-front` without a message link

### ISSUE THREAD

**Issue by FractalGeodes** [ijsbol/pk-utility-app#1]
Very cool and useful little thing!! I think we'll get a lot of use out of the fronter view especially, but I have a possible idea for something:

The fact that you _need_ to have a message url in the `check-front` command seems, odd to us. I feel like it could be useful if it does what I think it does (checking the front at the time that a certain message was sent) but in a lot of cases I can see myself just wanting to check to see who's there _right now_. Having to get a message link to put into the hungry command is a bit annoying for that purpose, I feel.

In other words, I'm requesting that the message link in `check-front` be optional, and to check the current front of the mentioned person when used. Alternatively, both could be made optional, and when neither is set it fetches the runner's fronters (use case could be, for example, quickly showing someone who's fronting)

I could probably have a look into making a PR for this myself sometime tomorrow, actually. I don't really have a lot else that I've been doing recently...

**Comment by ijsbol** [comment]
hiya, this is already a feature ^-^ we use message context apps for this - you need to right click and select the "apps" section then "Check front for message" (if on mobile, hold down on the message instead).

**Comment by FractalGeodes** [comment]
Ah yeah, we were aware of that!!! We just feel that having a way to do the textual command without needing to touch the mouse at all would be a lot more convenient, especially for the use case of showing front to someone easily

**Comment by ijsbol** [comment]
ooo okay i see, i like this idea! i'll get it added :)

**Comment by ijsbol** [comment]
This is now a feature :) Thank you for the suggestion!
